test/torch/tensors/test_additive_shared.py

```diff
@@ -802,6 +813,28 @@ def test_comp(workers, protocol, force_preprocessing):
     assert ((x > y).get().float_prec() == (t1 > t2)).all()
     assert ((x < y).get().float_prec() == (t1 < t2)).all()
 
+    # for x as AST and  y as FPT
```

Comment:
Q: Could you move those to another test function? (because this function would be called 2 times - one with protocol = "fss" and once with protocol "snn"). You can also parameterize that function with only "fss"

Reply:
Done